alby-nostr-wallet-connect/umbrel-app.yml

```diff
@@ -2,32 +2,32 @@ manifestVersion: 1
 id: alby-nostr-wallet-connect
 name: Nostr Wallet Connect
 tagline: The power of the zaps at the tip of your fingers.
-category: finance
-version: "0.3.0"
+icon: https://raw.githubusercontent.com/getAlby/umbrel-community-app-store/master/alby-nostr-wallet-connect/images/nwc_icon.svg
+category: Finance
+version: "0.4.1"
 port: 58000
 description: >-
-  Connect your LND node over Nostr to zap directly from within supported apps like Damus & Amethyst.
+  Connect your LND node over Nostr.
 
+  Zap directly from within supported apps like Damus & Amethyst.
 
-  🛠️ Instructions
+  Amethyst step-by-step guide: https://blog.getalby.com/native-zapping-in-amethyst
 
-  - Amethyst step-by-step guide: https://blog.getalby.com/native-zapping-in-amethyst
-
-  - Damus (currently only on Testflight) video guide: https://streamable.com/bareih
+  Damus (currently only on Testflight) video guide: https://streamable.com/bareih
 developer: Alby
 website: http://getalby.com/
 submitter: Alby
-submission: https://github.com/getumbrel/umbrel-apps/pull/636
+submission: https://github.com/getAlby/nostr-wallet-connect
 repo: https://github.com/getAlby/nostr-wallet-connect
 support: https://github.com/getAlby/nostr-wallet-connect/issues
 gallery:
-  - 1.jpg
-  - 2.jpg
-  - 3.jpg
+  - https://raw.githubusercontent.com/getAlby/umbrel-community-app-store/master/alby-nostr-wallet-connect/images/nwc_1.png
+  - https://raw.githubusercontent.com/getAlby/umbrel-community-app-store/master/alby-nostr-wallet-connect/images/nwc_2.png
+  - https://raw.githubusercontent.com/getAlby/umbrel-community-app-store/master/alby-nostr-wallet-connect/images/nwc_3.png
 releaseNotes: >-
-  This release includes a number of bug fixes and improvements. Full release notes are available at https://github.com/getAlby/nostr-wallet-connect/releases/tag/0.3.0
-dependencies:
-  - lightning
+  - This release adds the following Nostr Wallet Connect (NIP-47) extension methods: pay_keysend, make_invoice, lookup_invoice, list_transactions, get_balance, get_info which can be used by developers to make awesome apps powered by NWC.
+  - Improved app connection UI.
```

Comment:
With this yaml format, an extra blank newline is needed between the bullet points for them to show up on separate lines. Right now, this will look like:

<img width="1285" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/b6c459d9-bd4d-458f-a240-7ea6a60220af">